mobile/lib/utils/share_util.dart

```diff
@@ -261,3 +267,70 @@ Rect _sharePosOrigin(BuildContext? context, GlobalKey? key) {
   }
   return rect;
 }
+
+/// The [ScreenshotController] is used to capture the image of a widget that serves as
+/// a placeholder for an album link. The placeholder is then shared along with the album link.
+final ScreenshotController screenshotController = ScreenshotController();
```

Comment:
Better to create create an instance of `ScreenshotController` inside `shareAlbumLinkWithPlaceholder` and pass that `ScreenshotController` to `_createAlbumPlaceholder` from inside `shareAlbumLinkWithPlaceholder`.

Why: This variable now has a global scope, which is unnecessary in this case and could lead to issues in future. For example, someone could use this accidentally in another file.